sdk/cosmos/azure-cosmos/test/test_crud_subpartition.py

```diff
@@ -0,0 +1,453 @@
+        super(TimeoutTransport, self).__init__()
+
+    def send(self, *args, **kwargs):
+        if kwargs.pop("passthrough", False):
+            return super(TimeoutTransport, self).send(*args, **kwargs)
+
+        time.sleep(5)
+        if isinstance(self._response, Exception):
+            raise self._response
+        output = requests.Response()
+        output.status_code = self._response
+        response = RequestsTransportResponse(None, output)
+        return response
+
+
+@pytest.mark.usefixtures("teardown")
+class CRUDTests(unittest.TestCase):
+    """Python CRUD Tests.
+    """
+    configs = test_config._test_config
+    host = configs.host
+    masterKey = configs.masterKey
+    connectionPolicy = configs.connectionPolicy
+    last_headers = []
+
+    def __AssertHTTPFailureWithStatus(self, status_code, func, *args, **kwargs):
+        """Assert HTTP failure with status.
+
+        :Parameters:
+            - `status_code`: int
+            - `func`: function
+        """
+        try:
+            func(*args, **kwargs)
+            self.assertFalse(True, 'function should fail.')
+        except exceptions.CosmosHttpResponseError as inst:
+            self.assertEqual(inst.status_code, status_code)
+
+    @classmethod
+    def setUpClass(cls):
+        if (cls.masterKey == '[YOUR_KEY_HERE]' or
+                    cls.host == '[YOUR_ENDPOINT_HERE]'):
+            raise Exception(
+                "You must specify your Azure Cosmos account values for "
+                "'masterKey' and 'host' at the top of this class to run the "
+                "tests.")
+        cls.client = cosmos_client.CosmosClient(cls.host, cls.masterKey, connection_policy=cls.connectionPolicy)
+        cls.databaseForTest = cls.configs.create_database_if_not_exist(cls.client)
+
+    def test_collection_crud(self):
+        created_db = self.databaseForTest
+        collections = list(created_db.list_containers())
+        # create a collection
+        before_create_collections_count = len(collections)
+        collection_id = 'test_collection_crud ' + str(uuid.uuid4())
+        collection_indexing_policy = {'indexingMode': 'consistent'}
+        created_recorder = RecordDiagnostics()
+        created_collection = created_db.create_container(id=collection_id,
+                                                         indexing_policy=collection_indexing_policy,
+                                                         partition_key=PartitionKey(path=["/pk1", "/pk2", "/pk3"],
+                                                                                    kind="MultiHash"),
+                                                         response_hook=created_recorder)
+        self.assertEqual(collection_id, created_collection.id)
+        assert isinstance(created_recorder.headers, Mapping)
+        assert 'Content-Type' in created_recorder.headers
+        assert isinstance(created_recorder.body, Mapping)
+        assert 'id' in created_recorder.body
```

Comment:
```suggestion
        created_collection = created_db.create_container(id=collection_id,
                                                         indexing_policy=collection_indexing_policy,
                                                         partition_key=PartitionKey(path=["/pk1", "/pk2", "/pk3"],
                                                                                    kind="MultiHash"))
        self.assertEqual(collection_id, created_collection.id)
```
Please remove any uses of the old diagnostics file since we will be removing it and it is no longer relevant.